test/integration/scheduler/volume_binding_test.go

```diff
@@ -461,7 +461,7 @@ func makePod(name, ns string, pvcs []string) *v1.Pod {
 			Containers: []v1.Container{
 				{
 					Name:    "write-pod",
-					Image:   "k8s.gcr.io/busybox:1.24",
+					Image:   "gcr.io/google_containers/busybox:1.24",
```

Comment:
Why did you need to change this?

Reply:
See this PR: 
https://github.com/kubernetes/kubernetes/commit/e9dd8a68f60d2565069caed068f8fe4d493ea5ce#diff-c0800dd83834b3847acb7efd347d6d34

Reply:
Odd that the revert did not roll this back too?